tests/integration/common-operator-integ-suite.sh

```diff
@@ -14,17 +14,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# To be able to run this script it's needed to pass the flag --ocp or --kind
```

Comment:
put `set -euo pipefail` at the top of all scripts

Reply:
Moved, to the top but adding the -x flag also to trace all the commands `set -eux -o pipefail`